docs/volumes.adoc

```diff
@@ -1,155 +1,156 @@
-= Kadalu Storage Volumes
+= Kadalu Storage Pools
 
-=== Volume Creation
+Kadalu Storage pool is a collection of one or more export directories from multiple servers.
 
-Create a Kadalu Storage Volume with Storage units from multiple Storage nodes.
+=== Pool Creation
+
+Create a Kadalu Storage Pool with Storage units from multiple Storage nodes.
 
 [source,console]
 ----
-[root@server1 ~]# kadalu volume create DEV/vol1 server1.example.com:/exports/vol1/s1
-Volume vol1 created successfully
+[root@server1 ~]# kadalu pool create pool1 server1.example.com:/exports/pool1/s1
+Pool pool1 created successfully
 ID: 02b6a3a4-e704-47ee-bb05-e541a561a921
 ----
 
 Where
 
-- `DEV` is Pool name
-- `vol1` is the Volume name
+- `vol1` is the Pool name
 - `server1.example.com` is the node from which the Storage unit will be exported.
 - `/exports/vol/s1` is the mounted path of Storage Unit.
 
-Automatically create the Pool and add nodes to Pool that are specified in the Volume create request by adding `--auto-create-pool` and `--auto-add-nodes`.
+Automatically add nodes to the Cluster that are specified in the Pool create request by adding `--auto-add-nodes`.
```

Comment:
Here cluster refers to group of pools, or the pool itself?

Reply:
Cluster is group of nodes that can communicate each other. Pools are created using the nodes of a Cluster.